src/main/java/domain/GameResult.java

```diff
@@ -0,0 +1,46 @@
+package domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public enum GameResult {
+
+    WIN("승"),
+    LOSE("패"),
+    DRAW("무");
+
+    private final String description;
+
+    GameResult(final String description) {
+        this.description = description;
+    }
+
+    public static List<GameResult> getAllGameResults() {
+        return Arrays.stream(GameResult.values()).toList();
+    }
+
+    public static GameResult swapGameResult(final GameResult gameResult) {
+        if (Objects.equals(gameResult, WIN)) {
```

Comment:
```suggestion
        if (gameResult == WIN) {
```
enum은 위와 같이 비교가 가능합니다. 왜 그럴까요?

Reply:
Enum에서 상수는 단 하나의 인스턴스만 가지므로 ==는 같은 객체를 참조하고 있는지 확인하기 때문에 동일한 객체를 참조하고 있는지 확인이 가능합니다. equals도 마찬가지로 인스턴스가 하나이기 때문에 동일한 객체인지 확인하기 위해 ==를 사용하도록 오버라이딩이 되어있습니다. 따라서 equals를 사용하는것은 불필요한 메서드 호출입니다.